website/blog/2024-04-15-0.74-post.md

```diff
@@ -0,0 +1,407 @@
+        top: 25,
+        position: 'absolute',
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+
+<td width="300">
+
+![Static Example](../static/blog/assets/0.74-static-example.png)
+
+</td>
+
+</tr>
+
+</table>
+Notice how the green `View` declares `left` and `top` and it is positioned relative to the blue `View`, not its parent.
+
+`position: 'static'` may also affect [z-ordering](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) of views. Similar to how `position: 'static'` works with containing blocks, static will not be considered when forming stacking contexts. This allows for stacking an element relative to an ancestor that is not a direct parent.
+
+<table>
+
+<tr>
+
+<td width="350">
+
+```jsx
+<View
+  style={{
+    backgroundColor: 'blue',
+    width: 200,
+    height: 200,
+    flexDirection: 'row',
+  }}>
+  <View
+    style={{
+      backgroundColor: 'orange',
+      width: 50,
+      height: 50,
+      left: 10,
+      position: 'relative',
+      zIndex: 10,
+    }}
+  />
+  <View
+    style={{
+      backgroundColor: 'red',
+      width: 50,
+      height: 50,
+      position: 'static',
+    }}>
+    <View
+      style={{
+        backgroundColor: 'green',
+        width: 25,
+        height: 25,
+        zIndex: 11,
+        position: 'absolute',
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+
+<td width="300">
+
+![Static zIndex Example](../static/blog/assets/0.74-zindex-example.png)
+
+</td>
+
+</tr>
+
+</table>
+
+React Native continues to default to `position: 'relative'` when no `position` is set.
+
+## Preparing for the New Architecture
+
+- bridgeless by default?
+- link out to separate New Architecture update?
+- point to New Architecture landing page
+
+## Batched `onLayout` updates (New Architecture)
+
+State updates in `onLayout` callbacks are now batched. Prior, each state update in the `onLayout` event would result in a new render commit.
+
+```jsx
+function MyComponent(props) {
+  const [state1, setState1] = useState(false);
+  const [state2, setState2] = useState(false);
+
+  return (
+    <View>
+      <View
+        onLayout={() => {
+          setState1(true);
+       }}>
+      <View
+         onLayout={() => {
+          // When this event is executed, state1's new value is no longer observable here.
+          setState2(true);
+        }}>
+ 	    </View>
+    </View>
+  );
+}
+```
+
+In 0.74, `setState1` and `setState2` updates are batched together. This change is [expected behavior in React](https://react.dev/learn/queueing-a-series-of-state-updates#react-batches-state-updates) and allows for less re-renders.
+
+:::danger
+This change may break code that has relied on un-batched state updates. You'll need to refactor this code to use [updater functions](https://react.dev/learn/queueing-a-series-of-state-updates#updating-the-same-state-multiple-times-before-the-next-render) or equivalent.
+:::
+
+## Removal of deprecated `PropTypes`
```

Comment:
Can we have Removal & Deprecations at the end of this post?

Reply:
Is it worth to mention that this, at least in theory, shaved off 26.4kB from minified JS bundle? Devs like to read this kind of stats

Reply:
@huntie yes please let's include it